script.js

```diff
@@ -0,0 +1,38 @@
+const inputBox = document.getElementById("input-box");
+const listContainer = document.getElementById("list-container");
+
+function addTask() {
+  if (inputBox.value === '') {
+    alert("You must write something!");
+  }
+  else {
+    let li = document.createElement("li");
+    li.innerHTML = inputBox.value;
+    listContainer.appendChild(li);
+    let span = document.createElement("span");
+    span.innerHTML = "\u00d7";
+    li.appendChild(span);
+  }
+  inputBox.value = "";
+  saveData();
+}
+
+listContainer.addEventListener("click", function(e) {
+  if (e.target.tagName === "LI") {
+    e.target.classList.toggle("checked");
+    saveData();
+  }
+  else if (e.target.tagName === "SPAN") {
+    e.target.parentElement.remove();
+    saveData();
+  }
+}, false);
+
+function saveData() {
+  localStorage.setItem("data", listContainer.innerHTML);
```

Comment:
I don't think that storing the HTML in the localstorage is a good approach, poses a security vulnerability, this is prone to XSS attacks (script injection).

You should instead store the todo list instead, something like

```
[{name: "1st todo", checked: true},{name:"2nd todo",checked:false}]
```